app/src/main/java/cloudcity/util/CloudCityUtil.java

```diff
@@ -75,4 +75,21 @@ private static boolean sendIperf3Data(NetworkDataModel data) {
         requestData.add(data);
         return CloudCityHelpers.sendData(address, token, requestData);
     }
+
+    /**
+     * Method that rounds a double {@code numberToRound} to {@code numberOfDecimals} number of decimals and returns the
+     * corrected number
+     *
+     * @param numberToRound    the number to round to a fixed number of decimals
+     * @param numberOfDecimals the number of decimals to allow after the decimal point
+     * @return the fixed number
+     */
+    public static double roundToNumberOfDecimals(double numberToRound, int numberOfDecimals) {
+        // multiply by 10^numberOfDecimals, round, floor, then return divided
+        double multiplier = Math.pow(10, numberOfDecimals);
+        double basis = numberToRound * multiplier;
+        basis = Math.round(basis);
+        double retVal = basis / multiplier;
+        return retVal;
+    }
```

Comment:
lets solve those problems when we see them happening instead of potentially introducing new problems by avoiding "possible" problems which will not be happening with the current usecase.

Reply:
resolving.